src/renderer/features/proxies/workers/proxy-worker.ts

```diff
@@ -161,7 +161,7 @@ async function getProxies(
   }
 
   const proxiesToRemove = proxies.filter((p) => existingProxies.some((ep) => isEqual(p, ep)));
-  const proxiedesToRemove = Object.values(accounts)
+  const proxiedesToRemove = Object.values(proxiedes)
```

Comment:
Maybe `proxides`?

Reply:
we use `proxied` in [docs](https://docs.google.com/document/d/1xwzz-geiEvTxIPRPeAh_5R3PENuYjefzE5C-sRtaVtY/edit#heading=h.gfxjqyl4u0ai), looks like multiple should be `proxiedes`